qiskit_optimization/applications/optimization_application.py

```diff
@@ -54,7 +54,7 @@ def _result_to_x(self, result: Union[OptimizationResult, np.ndarray]) -> np.ndar
         else:
             raise TypeError(
                 "Unsupported format of result. Provide an　OptimizationResult or a",
-                "binary array using np.ndarray instead of {}".format(type(result)),
+                f"binary array using np.ndarray instead of {type(result)}",
```

Comment:
```suggestion
                f" binary array using np.ndarray instead of {type(result)}",
```
I think it ought to have had a space if this and the above string get concatenated otherwise it will read `or abinary` instead of of `or a binary`